ropetest/contrib/autoimporttest.py

```diff
@@ -142,6 +142,22 @@ def test_generate_full_cache(self):
         for table in self.importer._dump_all():
             self.assertTrue(len(table) > 0)
 
+    def test_generate_full_cache(self):
+        """The single thread test takes much longer than the multithread test but is easier to debug"""
+        single_thread = False
+        self.importer.generate_modules_cache(single_thread=single_thread)
+        """
+        Create a temporary directory and set permissions to 000
+        """
```

Comment:
These doesn't seem to be docstring, they should just be regular comment.

```suggestion
        # The single thread test takes much longer than the multithread test but is easier to debug
        self.importer.generate_modules_cache(single_thread=False)
        
        # Create a temporary directory and set permissions to 000
```